cyberark/conjur
# Adds a `conjur-dev` CLI

#### What does this pull request do?
This PR provides a new CLI tool `conjur-dev` in the `dev` folder.  This tool simplifies:
* Stepping into the running Conjur container
* Loading policies into Conjur

#### What background context can you provide?
This is part of ongoing work to improve the development experience, and make it easier to develop and test Conjur.

#### Where should the reviewer start?
`README.md` has most of the details, `dev/conjur-dev` has the fun stuff.

#### How should this be manually tested?
Pull this branch down and start a development environment.  In a new shell, you should be able to run the following:
* `./conjur-dev --help`
* `./conjur-dev exec`
* `./conjur-dev policy load cucumber dev/files/authn-ldap/policy.yml` # assuming there is a policy file in that location.
